tests/rundb/test_httpdb.py

```diff
@@ -633,3 +633,70 @@ def _assert_projects(expected_project, project):
     )
     assert expected_project.spec.desired_state == project.spec.desired_state
     assert expected_project.spec.desired_state == project.status.state
+
+
+def _assert_source_lists_match(db, expected_response):
+    response = db.list_marketplace_sources()
+
+    exclude_paths = [
+        "root['source']['metadata']['updated']",
+        "root['source']['metadata']['created']",
+    ]
+    for i in range(len(expected_response)):
+        assert (
+            deepdiff.DeepDiff(
+                expected_response[i].dict(),
+                response[i].dict(),
+                exclude_paths=exclude_paths,
+            )
+            == {}
+        )
+
+
+def test_marketplace(create_server):
```

Comment:
Can you please move this test to be under `/tests/integration/sdk_api/marketplace/test_marketplace.py`
Eventually the tests in this file should go under that module, I will handle the refactor at some point, but let's put new tests there